modules/seqexec/server/src/main/scala/seqexec/server/altair/AltairKeywordReader.scala

```diff
@@ -82,14 +81,12 @@ object AltairKeywordReaderEpics extends AltairKeywordReaderLUT {
   def apply[F[_]: Sync](sys: AltairEpics[F]): AltairKeywordReader[F] = new AltairKeywordReader[F] {
 
     override def aofreq: F[Double] =
-      Nested(sys.aoexpt)
-        .filter(_ =!= 0.0f)
-        .map(1 / _.toDouble)
-        .value
+      sys.aoexpt
+        .map(1.0 / _.toDouble)
```

Comment:
Why is the filter off? Are you assuming that `safeValOrDefalt` would catch a divide by zero?

Reply:
Yes